WordPress/Sniffs/NamingConventions/ValidVariableNameSniff.php

```diff
@@ -57,6 +57,15 @@ class WordPress_Sniffs_NamingConventions_ValidVariableNameSniff extends PHP_Code
 	);
 
 	/**
+	 * Mixed-case variables used by WordPress.
+	 *
+	 * @var array
+	 */
+	public $wordpress_mixed_case_vars = array(
+		'EZSQL_ERROR',
```

Comment:
1. `public` properties within sniffs can be overruled by setting the property in the ruleset. I think the visibility here needs to change to either `protected` or `private`.
2. The pattern of `array( 'string_key' =>true )` for array properties would allow for checking using `isset()` rather than the slower `in_array()`.